cpp/src/link_prediction/jaccard_impl.cuh

```diff
@@ -27,19 +27,13 @@ struct jaccard_functor_t {
   template <typename weight_t>
   weight_t __device__ compute_score(weight_t cardinality_a,
                                     weight_t cardinality_b,
-                                    weight_t cardinality_a_intersect_b) const
+                                    weight_t cardinality_a_intersect_b,
```

Comment:
I feel like a different name than `cardinality_X` might be appropriate for these.  `cardinality` defines the size of a set.  This is correct for the unweighted case.  But for the weighted case the things being passed in are a function on the weights of the incident edges.  Using the word `cardinality` to describe that seems incorrect.

I had used `weight_X` in the weighted variant that I sketched out... although I'm not sure `weight` is really a better word either.

Reply:
weight_X sounds better, it's more appropriate for SG an MG. Will be renamed in next commit.